README.md

```diff
@@ -3,7 +3,7 @@
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/magento/magento2?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 [![Crowdin](https://d322cqt584bo4o.cloudfront.net/magento-2/localized.svg)](https://crowdin.com/project/magento-2)
 <h2>Welcome</h2>
-Welcome to Magento 2 installation! We're glad you chose to install Magento 2, a cutting-edge, feature-rich eCommerce solution that gets results.
+Welcome to Magento 2 installation! We're glad you chose to install Magento 2, a cutting-edge, feature-rich E-Commerce solution that gets results.
```

Comment:
As I see in other modules the correct form that we usually use is e-commerce instead of E-Commerce
```suggestion
Welcome to Magento 2 installation! We're glad you chose to install Magento 2, a cutting-edge, feature-rich E-Commerce solution that gets results.
```